.github/workflows/tests_secondary.yml

```diff
@@ -274,7 +274,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [ubuntu-latest, macos-latest, windows-latest]
+        runs-on: [ubuntu-latest]
```

Comment:
Why do we need to test chromium-next on windows and macOS, but not headless-shell? it's technically a new binary as well, are we sure it behaves the same?

Reply:
We test `headless-shell` now on every bot, since it's a part of regular `chromium`. On the other hand, `chromium-next` is only tested on the new bots.